app/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#-------------------------------------------------------------------------------
+# Zephyr Example Application
+
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(app LANGUAGES C VERSION 1.0.0)
+
+#-------------------------------------------------------------------------------
+# Options
```

Comment:
More comments on why CMAKE_EXPORT_COMPILE_COMMANDS is set would be useful.

Maybe some commented-out common use cases, like EXTRA_CFLAGS?

Reply:
> What common options do you suggest?

Maybe start with this:

```cmake
# EXTRA_CFLAGS lets you add compiler flags. For example, to get colored
# output from GCC, uncomment this:
#
# set(EXTRA_CFLAGS "-fdiagnostics-color=always")
```

We can add more examples depending on what support requests roll in.

Reply:
simplified CMake, no options are present now